cocotb/binary.py

```diff
@@ -69,7 +69,7 @@ class BinaryValue(object):
     _resolve_to_0    = "xXzZuU"  # noqa
     _permitted_chars = "xXzZuU" + "01"  # noqa
 
-    def __init__(self, value=None, bits=None, bigEndian=True,
+    def __init__(self, value=None, bits=None, bigEndian=False,
```

Comment:
Yeah, that makes sense.

I've got another fix for little endian in __getitem__
Let me add that in and give you another pull request, without the change in default behaviour.